src/main/resources/screens/live/APPROVE_MINISTER_SIGN_OFF.json

```diff
@@ -9,6 +9,10 @@
       ],
       "props": {
         "choices": [
+          {
+            "label": "Yes (Legacy)",
```

Comment:
Does this need adding to the accordion config in hocs-casework?

Reply:
It will do - I'll follow up with that shortly - just want to get this in.